tools/SchemaManager/Commands/CurrentCommand.cs

```diff
@@ -33,7 +34,7 @@ public static async Task HandlerAsync(InvocationContext invocationContext, Uri s
             {
                 // Base schema is required to run the schema migration tool.
                 // This method also initializes the database if not initialized yet.
-                await BaseSchemaRunner.EnsureBaseSchemaExistsAsync(connectionString);
+                await BaseSchemaRunner.EnsureBaseSchemaExistsAsync(connectionString, cancellationToken);
```

Comment:
Shouldn't `EnsureInstanceSchemaRecordExists` below be async too?

Reply:
It doesn't really have any async call, hence I left it like this.